README.md

```diff
@@ -92,7 +92,7 @@ Measuring the Web Vitals scores for your real users is a great first step toward
 
 The "attribution" build helps you do that by including additional diagnostic information with each metric to help you identify the root cause of poor performance as well as prioritize the most important things to fix.
 
-The "attribution" build is slightly larger than the "standard" build (by about 600 bytes, brotli'd), so while the code size is still small, it's only recommended if you're actually using these features.
+The "attribution" build is slightly larger than the "standard" build (by about 1.5K, brotli'd), so while the code size is still small, it's only recommended if you're actually using these features.
```

Comment:
Oof. That's gotta hurt!

Reply:
I'm pretty sure this difference has built up over time and wasn't all in these latest changes. Like, I don't remember updating this number after the INP attribution changes in v4.